packages/EasyActivity/src/Bridge/Symfony/DependencyInjection/EasyActivityExtension.php

```diff
@@ -42,6 +45,13 @@ public function load(array $configs, ContainerBuilder $container): void
         if ($this->easyDoctrineBundleIsRegistered($container)) {
             $loader->load('easy-doctrine-bridge-services.php');
         }
+
+        $circularReferenceHandler = $config['circular_reference_handler'];
```

Comment:
Can we add the definition for `CircularReferenceHandlerInterface::class` in `packages/EasyActivity/src/Bridge/Symfony/Resources/config/services.php` as for other services?

Reply:
Circular handler is related on  application and may not be necessary default `CircularReferenceHandler`. So in `EasyActivityExtension` we get handler class from config and make it as service dynamically.

Reply:
I agree with @mvlad on this, and we already have examples in other packages so keeping things consistent would be great.
I agree with your point too @skrut , application might want to have their own CircularReferenceHandler and this will be able to do so.

- Create a unique service id for the CircularReferenceHandler of this package
- Register the default implementation in the `config/services.php` using this service id

Done! 😄 

If an application wants to customize this logic, then they simply do the same thing, create their own implementation and register it using the service id from the package, this will automatically replace the default CircularReferenceHandler with theirs.

Reply:
Done

Reply:
@skrut, it seems we can remove the `circular_reference_handler` setting now. Also please replace `'easy_activity.circular_reference_handler'` with `BridgeConstantsInterface::PARAM_CIRCULAR_REFERENCE_HANDLER` in the `services.php`.

Reply:
@mvlad, of course, thank you!